protocol/tx.go

```diff
@@ -10,17 +10,23 @@ import (
 	"chain/protocol/validation"
 )
 
-// ValidateTxCached checks a cache of prevalidated transactions
-// before attempting to perform a context-free validation of the tx.
-func (c *Chain) ValidateTxCached(tx *bc.Tx) error {
-	// Consult a cache of prevalidated transactions.
-	err, ok := c.prevalidated.lookup(tx.ID)
-	if ok {
+// ErrBadTx is returned for transactions failing validation
+var ErrBadTx = errors.New("invalid transaction")
+
+// ValidateTx validates the given transaction. A cache holds
+// per-transaction validation results and is consulted before
+// performing full validation.
+func (c *Chain) ValidateTx(tx *bc.TxEntries) error {
```

Comment:
It's not so much a rename as it is part of a brand-new, simpler interface to validation that happens to look a lot like the old one. See the discussion at https://github.com/chain/chain/pull/788#discussion_r108548333

`protocol.Chain` has no tx-validation entrypoint that isn't cached, so there's no need to include `Cached` in the name.